community/kernel/src/test/java/org/neo4j/unsafe/impl/batchimport/input/csv/CsvInputTest.java

```diff
@@ -488,6 +488,27 @@ public void shouldParsePointPropertyValues() throws Exception
         }
     }
 
+    @Test
+    public void shouldParsePointPropertyValuesWithCRSInHeader() throws Exception
+    {
+        // GIVEN
+        DataFactory data = data(
+                ":ID,name,point:Point{crs:WGS-84-3D}\n" +
+                        "0,Johan,\" { height :0.01 ,longitude:5, latitude : -4.2 } \"\n" );
+        Iterable<DataFactory> dataIterable = dataIterable( data );
```

Comment:
Another good test would be with WGS-84 in the header and x and y in the 1st row. Then we should see it chooses WGS-84 instead of Cartesian.

Reply:
Good idea!